build/csslint/xfail.list

```diff
@@ -55,6 +55,7 @@ apps/settings/style/settings.css 0 9
 apps/settings/style/settings_large.css 1 1
 apps/settings/style/screen_lock.css 0 2
 apps/settings/style/downloads.css 0 6
+apps/sms/views/conversation/style/message.css 2 0
```

Comment:
maybe you can try to contribute to csslint to make the tool know about margin-inline-start and margin-inline-end (and their friends) ?